sbol2/__init__.py

```diff
@@ -24,7 +24,7 @@
 from .implementation import Implementation
 from .interaction import Interaction
 from .location import Location, Range, Cut, GenericLocation
-from .measurement import Measurement
+from .measurement import Measure
```

Comment:
For consistency's sake, maybe go ahead an rename measurement.py to measure.py?

Reply:
I reverted the renaming of Measurement to Measure.